2.adoc

```diff
@@ -72,5 +72,165 @@ Optional description
+
+image::https://i.imgur.com/1uSz1xM.png[]
+
+* Vkladanie obrázkov
+  ** cez menu (otvorí sa nativny dialog), klávesovou skratkou (snippet) alebo drag&drop
+  ** možnosť automatického uploadu obrázku na server (nutné nainštalovať ďalšiu aplikáciu)
+
+#### Zajímavé features
+
+(viac menej asi popísané vyššie)
+
+* Pekný syntax highlighting kódu
+* Generovanie Table of Contents v dokumente
+* Themes
+  ** Na výber z niekoľkých tém, ktoré menia vzhľad ako aj vyrenderovaného dokumentu tak aj zdrojového kódu
+* Export do PDF, docx, OpenOffice, LaTeX, MediaWiki, Epub
+
+==== Shrnutí
+
+===== Plusy
+* Príjemný vzhľad, možnosť zmeny vzhľadu
+* live rendering šetrí miesto
+* prijemná práca s tabuľkami
+* vkladanie obrázkov drag&drop
+* podpora tabov
+
+===== Mínusy
+* Málo možnosti konfigurácie
+* Spellchecker nepodtrháva nesprávne slovo
+* Ak užívateĽ nepozná klávesové skratky je nutné siahnuť po myši (chýba command palleta)
+* komerčný editor
+* Nemožnosť nastaviť otvárenie nového tabu namiesto okna
+
+
+=== AsciiDocLIVE
+
+* https://asciidoclive.com
+* online open-source asciiDoc-only editor
+
+==== Porovnání rozdílů
+
+Jedná sa o webovú komponentu, ktorá ale nie je embedovateľná. Podporuje iba jazyk AsciiDoc. Kvôli chýbajúcej command pallete vyžaduje používanie myši ak si užívateľ nepamätá potrebnú klávesovú skratku. Podporuje zmenu vzhľadu pomocou css šablón. Dokument je možné uložiť na Dropbox, Google Drive alebo na filesystém. 
+
+==== Zhodnocení UI
+
+image::https://i.imgur.com/O8bixCn.png[]
+
+* Okno editoru je rozdelené na dve časti
+  ** zdrojový kód so zýraznenou syntaxou (len farebné rozlíšenie)
+  ** vyrenderovaný dokument (live rendering)
+  ** možnosť zapnúť synchronizované scrollovanie v nastaveniach
+
+
+* Klavesové skratky
+  ** možnosť vybrať z 3 presetov (default, vim, emacs)
+  ** UX fail: užívateľ nikde nezistí aké konkrétne klavesové skratky je možné používať 
+
+
+==== Pozorování UI v kontextu markup-editor
+
+* má Toolbar(menu) v hornej časti
+  ** Otvorenie dokumentu (z Dropboxu, GG Drive, filesystem) alebo vytvorenie nového dokumentu
+  ** uloženie (GG Drive, Dropbox, filesystem)
+  ** Nastavenia (sync scrollovanie, autosave, zmena šablóny, písma, presety klav. skratiek)
+  ** FAQ, About, AsciiDoc cheatsheet
+
+image::https://i.imgur.com/YzpaE45.png[]
+
+* Chýba command paleta
+* Nie je k dispozícií zoznam klávesových skratiek
+* Nie je možné prepínať režimy zobrazenia
+  ** jediná možnosť je mať naľavo zdrojový kód a napravo vyrenderovaný dokument :)
+  ** Žiadny distraction free mod, pri fullscreene stále vidno prvky prehliadača
+* Chýba statusbar
+  ** Chýba informácia o pozícií kurzora v dokumente
+* Nemá outline
+* Práca sa tabuľkami a obrázkami len na úrovni zdrojového kódu
+
+==== Zajímavé features
+
+* Themes
+  ** Na výber z niekoľkých tém, ktoré menia zvýraznenie zdrojového kódu
+  ** možnosť editovať šablónu
+
+==== Shrnutí
+
+===== Plusy
+
+- Jednoduchý
+- AsciiDoc?
```

Comment:
Co to znamena? Ze podporuje ADoc a ze jich je malo nebo...? Pokud jo, tak by to chtelo nejak vice self-descriptive.

Reply:
len také rypnutie do asciidocu :D otaznik umažem

Reply:
aha tak rýpnutie je v mínusoch, tu som chcel vyzdvihnúť že to vie asciidoc (a nie prehajpovany md)

Reply:
> len také rypnutie do asciidocu

Markdown kterej nema temer zadnej standard, bambiliardu pluginu a graziliardu interpretu, kterej si to kazdej vyklada po svym na tebe! :D